Frontend/src/app/errorpages/error404/error404.component.html

```diff
@@ -0,0 +1,29 @@
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>404</title>
+  <link href="https://fonts.googleapis.com/css2?family=Nunito:wght@300;400;600;700;800&display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+<div id="error">
```

Comment:
not adaptive
![image](https://user-images.githubusercontent.com/75213424/154852832-a55fee86-e482-437a-8c71-2bc24bd8d50f.png)